frontend/src/app/projects/models/backup-settings/job-run-overview/job-run-overview.component.ts

```diff
@@ -30,26 +43,91 @@ export class JobRunOverviewComponent implements OnInit {
 
   ngOnInit() {
     combineLatest([
-      this.projectService.project.pipe(filter(Boolean)),
-      this.modelService.model.pipe(filter(Boolean)),
-      this.pipelineService.pipeline.pipe(filter(Boolean)),
+      this.projectService.project.pipe(
+        filter(Boolean),
+        map((project) => project.slug)
+      ),
+      this.modelService.model.pipe(
+        filter(Boolean),
+        map((model) => model.slug)
+      ),
+      this.pipelineService.pipeline.pipe(
+        filter(Boolean),
+        map((pipeline) => pipeline.id)
+      ),
     ])
       .pipe(
         untilDestroyed(this),
-        switchMap(([project, model, pipeline]) =>
+        switchMap(([projectSlug, modelSlug, pipelineID]) =>
           this.pipelineRunService.loadPipelineRuns(
-            project.slug,
-            model.slug,
-            pipeline.id
+            projectSlug,
+            modelSlug,
+            pipelineID,
+            1,
+            this.pageSize
           )
         )
       )
       .subscribe();
   }
```

Comment:
As far as I can see this can be simplified a lot (probably I am missing something about the pagination). Here would be my simplified version:
```typescript
combineLatest([
      this.projectService.project.pipe(filter(Boolean)),
      this.modelService.model.pipe(filter(Boolean)),
      this.pipelineService.pipeline.pipe(filter(Boolean)),
    ])
      .pipe(untilDestroyed(this))
      .subscribe(([project, model, pipeline]) =>
        this.pipelineRunService.loadPipelineRuns(
          project.slug,
          model.slug,
          pipeline.id,
          1,
          this.pageSize
        )
      );
```

Changes I applied here:
- Do not use `map` here as IMO the mapping here makes this way harder to read
- Move the `switchMap` part into the `subscribe` as nothin is done with the result of `loadPipelineRuns` (and also `loadPipelineRuns` should not return anything -> having `void` as return type)